src/entities/Entity.ts

```diff
@@ -138,12 +144,19 @@ export class Entity<T> {
 
     private signalEntity(
         returnState: EntityState,
+        request: RequestMessage,
         entity: EntityId,
         operationName: string,
         operationInput?: unknown
     ): void {
         returnState.signals.push(
-            new Signal(entity, operationName, operationInput ? JSON.stringify(operationInput) : "")
+            new Signal(
+                entity,
+                operationName,
+                operationInput ? JSON.stringify(operationInput) : "",
+                request.id,
+                new Date().getTime()
```

Comment:
Does this match how we are saving the time in the WebJobs extension (i.e. DateTimeOffset)?

Reply:
Yep yep, the method description is "Returns the stored time value in milliseconds since midnight, January 1, 1970 UTC." which is exactly what the WebJobs side expects when it calls `DateTimeOffset.FromUnixTimeMilliseconds` on this field value

Reply:
Great!